src/mesh/generated/meshtastic/mesh.pb.h

```diff
@@ -59,6 +59,8 @@ typedef enum _meshtastic_HardwareModel {
     meshtastic_HardwareModel_TLORA_T3_S3 = 16,
     /* B&Q Consulting Nano G1 Explorer: https://wiki.uniteng.com/en/meshtastic/nano-g1-explorer */
     meshtastic_HardwareModel_NANO_G1_EXPLORER = 17,
+    /* CanaryRadio.io LLC Handheld https://canaryradio.io */
```

Comment:
Canary Radio Company

Reply:
This is a generated file and will be fixed in the protobuf repository